### PR TITLE
T36: Build Profile Page

### PR DESCRIPTION
**Pull request by suzunak**
Added the following folder and files:
- Profile
  - profile.jsx
  - profile.module.css

Edits under App.js:
- Added route for profile page

Edits under navigation.js:
- Added link to profile page

Image of profile page:

ss_profile_page

### CODE REVIEW

File: frontend/src/components/Profile/profile.jsx
Comment (justinpchen94) [inline frontend/src/components/Profile/profile.jsx:1]: nit: capitalize to `Profile.jsx`

File: frontend/src/components/Profile/profile.module.css [hunk 1]
```diff
@@ -0,0 +1,154 @@
+.profile-page {
+    display: grid;
+    height: 100%;
+    width: 100%;
+    background-color: rgb(254, 239, 221);
```
Comment (justinpchen94) [inline frontend/src/components/Profile/profile.module.css:5]: i forget, have we defined css variables yet for our common colors?

**Review by justinpchen94 — APPROVED** [review]
👏

File: frontend/src/components/Profile/profile.module.css (hunk 1 above)
Comment (suzunak) [reply frontend/src/components/Profile/profile.module.css:5]: We haven't defined css variables for our colors. Should I create them?

Comment (justinpchen94) [reply frontend/src/components/Profile/profile.module.css:5]: can't hurt!

**Pull request merged**